roles/mkcert/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+- name: Install mkcert and nss
+  homebrew:
+    name:
+      - mkcert
+      - nss
+    state: "{{ homebrew_package_state }}"
+
+- name: Install certifi
+  pip:
+    name: certifi
+    state: latest
+
+- name: Setup mkcert
+  shell: mkcert -install
+  become: yes
+
+# See https://github.com/Homebrew/homebrew-core/blob/master/Formula/openssl%401.1.rb#L54
+- name: Force generation of OpenSSL CA file from the system keychain
+  shell: brew postinstall openssl
+
+- name: Include mkcert CA in OpenSSL CA
+  shell: cat "$(mkcert -CAROOT)/rootCA.pem" >> {{ openssl_ca_file }}
+
+- name: Replace certifi CA 
+  shell: cp {{ openssl_ca_file }} $(python -m certifi)
```

Comment:
not working if python2 is installed -> https://github.com/TheGrowingPlant/mac-playbook/issues/91